pkg/secrets/vault.go

```diff
@@ -75,6 +75,12 @@ func (c *vaultClient) setItemAtPath(path, field string, content string) error {
 	return c.upstream.UpsertKV(path, data)
 }
 
+func (c *vaultClient) HasItem(itemName string) bool {
+	path := c.pathFor(itemName)
+	_, err := c.upstream.GetKV(path)
+	return err != nil
```

Comment:
Please make this a IsNotFound check instead

Reply:
updated.